ajax/libs/spectragram/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spectragram",
+  "filename": "spectragram.min.js",
+  "version": "1.0.3",
+  "description": "An easy jQuery plugin for Instagram API to fetch and display user, popular or tags photo feeds inside your web application or site.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/adrianengine/jquery-spectragram.git"
+  },
+  "keywords": [
+    "jquery-plugin",
+    "ecosystem:jquery",
+    "gallery",
+    "instagram",
+    "photos",
+    "spectragram"
+  ],
+  "author": "Adrian Quevedo <npm@adrianquevedo.com> (http://adrianquevedo.com/)",
+  "license": "MIT",
+  "homepage": "https://spectragram.js.org/",
+  "npmName": "spectragram",
```

Comment:
There is officially minified file on GitHub for each version but npm not. Let's use git auto-update config here. Thanks.